### ISSUE TITLE
advanced search results alignment is off

### ISSUE THREAD

**Issue by SDavisMedia** [easydigitaldownloads/vendd#162]
http://glui.me/?i=h935p7r55frgnm3/2016-02-25_at_1.17_AM.png/

**Comment by SDavisMedia** [comment]
This is not a CSS issue. I believe it has to with the order of the search results returned (posts vs pages vs downloads). The layout expects a certain display order every time. Dangit.

**Comment by spencerfinnell** [comment]
It would probably make the most sense to sort them all in PHP first: https://gist.github.com/spencerfinnell/13705d5043a54274669d

Then you can just build out the 3 sections and choose right where to output each type.

**Comment by SDavisMedia** [comment]
@spencerfinnell Bingo! This is exactly what I was thinking. Thanks for the head start. Gonna play with this later this week and see if I can get it all set in stone.

**Comment by SDavisMedia** [comment]
Went a slightly different route, @spencerfinnell. You should check it out. Pretty straight forward and something I'll probably do regularly moving forward with other themes.

**Comment by spencerfinnell** [comment]
Nice! Would have been my other choice.

If you really wanted to get crazy you could filter the main query with pre_get_posts to only return the post post type. So you're only doing two custom queries instead of 3.

**Comment by SDavisMedia** [comment]
@spencerfinnell Definitely did go that route for a while before pushing it up. Decided not to stick with it because of past experiences I've had with other plugins/themes screwing with `pre_get_posts`. Figured it'd be a little easier on me in the future if I kept all of advanced search results functionality separate from the default search before.

Likewise, it's kind of cool having the query args right there for each post type because I'll probably limit the results of each (for uniformity) and add a link or something for the full result list, you know?
